lib/express_validator.js

```diff
@@ -23,7 +23,7 @@ function ValidatorChain(param, failMsg, req, location, options) {
   this.validationErrors = [];
   this.failMsg = failMsg;
   this.req = req;
-
+  this.lastError = undefined; // used by withMessage to get the values of the last error
```

Comment:
Let's initialize this as `null`. It's kind of paradoxical to define something as `undefined`.